WordPress/src/main/java/org/wordpress/android/push/GCMMessageService.java

```diff
@@ -59,7 +59,7 @@
 
 public class GCMMessageService extends GcmListenerService {
     private static final ArrayMap<Integer, Bundle> sActiveNotificationsMap = new ArrayMap<>();
-    private static NotificationHelper sNotificationHelpers;
+    private static NotificationHelper sNotificationHelpers = new NotificationHelper();
```

Comment:
This can be `final`, and we can drop any null checks for `sNotificationHelpers` in this class.